### PR TITLE
chore(Kafka): change default Kafka container image

### PR DESCRIPTION
**Pull request by guidoiaquinti**
## Problem
See https://github.com/PostHog/posthog/issues/9080

## Changes

Change the default Kafka container image from the generic `wurstmeister/kafka:latest` to `bitnami/kafka:2.8.1-debian-10-r99`. From a Kafka version perspective, this change should be a no-op as the current [wurstmeister/kafka:latest](https://hub.docker.com/layers/wurstmeister/kafka/latest/images/sha256-4b63ea74fd3c380a4adca3cdda05db6467904f5d203b9ca10c0862ebaa662cba?context=explore) is built against 2.8.1.

We are picking `bitnami/kafka:2.8.1-debian-10-r99` as it is the latest default image for the upstream `bitnami/kafka` Helm chart before a breaking release.

## How did you test this code?
CI and local development are ✅  

I've tested the local development workflow by running:

1. switched branch to `master`
1. removed all containers & volumes running via: `docker rm -f $(docker ps -a -q) && docker volume rm $(docker volume ls -q)`
1. started external services via: `docker compose -f docker-compose.arm64.yml up zookeeper kafka clickhouse db redis`
1. executed the database migrations via: `DEBUG=1 python manage.py migrate && DEBUG=1 python manage.py migrate_clickhouse` 
1. started the service via `./bin/start`
1. verified everything is ✅  
1. switched branch to `kafka_upgrade_to_2_8_1`
1. stop/start external services via: `docker compose -f docker-compose.arm64.yml up zookeeper kafka clickhouse db redis`
1. started the service via `./bin/start`
1. verified everything is ✅

### CODE REVIEW

**Comment by guidoiaquinti** [comment]
@fuziontech not sure if we have e2e testing for Hobby deployment upgrade but let me know if this looks good to you. Thank you!

File: docker-compose.arm64.yml [hunk 1]
```diff
@@ -43,14 +43,19 @@ services:
         image: zookeeper
         restart: always
     kafka:
-        image: wurstmeister/kafka
+        image: bitnami/kafka:2.8.1-debian-10-r99
```
[the remark this answers is not in the document]
Comment (guidoiaquinti) [reply docker-compose.arm64.yml:46]: Fair point. Works 👍 on my M1

**Review by hazzadous — COMMENTED** [review]
lgtm

🤞 the hobby upgrade path is good

**Comment by guidoiaquinti** [comment]
> lgtm
> 
> 🤞 the hobby upgrade path is good

I've tested it now with:

1. setup a new DO droplet
1. run `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/posthog/posthog/HEAD/bin/deploy-hobby)"` as [per documentation](https://posthog.com/docs/self-host/deploy/hobby)
1. verified everything is ✅ 
1. run `cd posthog && git checkout kafka_upgrade_to_2_8_1` to update the checkout to this branch
2. run `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/posthog/posthog/HEAD/bin/upgrade-hobby)"` as [per documentation](https://posthog.com/docs/self-host/deploy/hobby)
1. verified everything is ✅ 

I'm going to merge this

**Pull request merged**